src/pages/viewer/Viewer.svelte

```diff
@@ -86,7 +89,7 @@
     <link rel="canonical" href={$viewer.document.canonicalUrl} />
 
     {#if $viewer.document && $viewer.document.noindex}
-      <meta name="robots" content="noindex">
+      <meta name="robots" content="noindex" />
```

Comment:
Browsers may be forgiving of this but `<meta>` tags are [void elements](https://developer.mozilla.org/en-US/docs/Glossary/Void_element) like `<input>` tags and technically aren't allowed to have closing tags.

Reply:
I think that's just formatting. Svelte is going to render it with `document.createElement` regardless of how it's written here.

That said, we should add a `.prettierrc` file so we can standardize what gets changed and how.